### PR TITLE
Disable "noKeyboard" logic for IE11

### LINKED ISSUE #3757: IE11: Send box is disabled after pressing ESCAPE from transcript
## Version

`4.12.0-master.873a827`

## Describe the bug

In IE11, given focus on the transcript, pressing <kbd>ESC</kbd> will focus on the send box. But can't start typing anything.

## Steps to reproduce

1. Press <kbd>SHIFT</kbd> <kbd>TAB</kbd> until focusing on the transcript (or terminator)
1. Press <kbd>ESCAPE</kbd>, focus should now in the send box (caret in the send box)
1. Press <kbd>A</kbd> to start typing a message

## Expected behavior

In the send box, it will display `A`, as what we entered.

Instead, despite the focus is in the send box, press any keys will not work.

## Workaround

End-user need to press <kbd>TAB</kbd> followed by <kbd>SHIFT</kbd> <kbd>TAB</kbd> to focus back on the send box before it start working again.

## Additional context

It works on Edge (Chromium).

Press <kbd>BKSP</kbd> in the "disabled-but-focused" send box, will cause page back navigation.

[Bug]

### LINKED ISSUE #3901: IE11: SVG icons should not be focusable
## Screenshots

After pressing <kbd>SHIFT</kbd> <kbd>TAB</kbd> once, `document.activeElement` indicates it is focusing on the `<svg>`.

Put this line in the console to dump the current focus:

```js
setInterval(function () { console.log(document.activeElement); }, 1000);
```

image

## Version

`4.13.0`

## Describe the bug

In IE11, press <kbd>SHIFT</kbd> <kbd>TAB</kbd> from the send box will focus on the `<svg>`, instead of the `<button>`.

## Steps to reproduce

1. Launch Internet Explorer 11
1. Go to https://compulim.github.io/webchat-loader/
1. Press <kbd>SHIFT</kbd> <kbd>TAB</kbd>, note the focus is on the icon of the upload button
1. Press <kbd>SHIFT</kbd> <kbd>TAB</kbd> again, note the focus is on the upload button
1. Press <kbd>SHIFT</kbd> <kbd>TAB</kbd> again, it is now on the transcript

## Expected behavior

It should not focus on step 3.

## Additional context

[Bug]

### PR DESCRIPTION
**Pull request by compulim**
> Fixes #3757. Fixes #3901.

### Fixes

-  Fixes [#3757](https://github.com/microsoft/BotFramework-WebChat/issues/3757). IE11: Send box should not be disabled after pressing <kbd>ESCAPE</kbd> key in the transcript, by [@compulim](https://github.com/compulim), in PR [#3902](https://github.com/microsoft/BotFramework-WebChat/pull/3902)
-  Fixes [#3901](https://github.com/microsoft/BotFramework-WebChat/issues/3901). IE11: SVG icons for buttons should not be focusable, by [@compulim](https://github.com/compulim), in PR [#3902](https://github.com/microsoft/BotFramework-WebChat/pull/3902)

### IE11: <kbd>ESCAPE</kbd> key should not disable send box

After pressing <kbd>ESCAPE</kbd> key while focused on the transcript, IE11 will disable the send box.

This is because IE11 did not pick up `readonly` attribute change after `focus()` is called.

The `noKeyboard` flag, which set the `readonly` attribute, is disabling soft keyboard on mobile devices.

Since IE11 don't run on mobile device, we can skip ignore the `noKeyboard` flag on IE11.

### IE11: Should not focus on `<svg>` icons

(Adopted from [this StackOverflow answer](https://stackoverflow.com/questions/29393144/how-to-prevent-svg-elements-from-gaining-focus-with-tabs-in-ie11))

IE11 by default assume the `<svg>` is focusable. Since `tabindex` attribute is not supported in `<svg>` when IE11 implement it, we are using the `focusable` attribute instead.

## Specific Changes

- Skip `noKeyboard` flag for IE11
- Set `focusable={true}` on all SVGs, they are icons for buttons

-  [x] I have added tests and executed them locally
-  [x] I have updated `CHANGELOG.md`
-  [x] ~I have updated documentation~

## Review Checklist

> This section is for contributors to review your work.

-  [x] ~Accessibility reviewed (tab order, content readability, alt text, color contrast)~
-  [x] Browser and platform compatibilities reviewed
-  [x] ~CSS styles reviewed (minimal rules, no `z-index`)~
-  [x] ~Documents reviewed (docs, samples, live demo)~
-  [x] ~Internationalization reviewed (strings, unit formatting)~
-  [x] ~`package.json` and `package-lock.json` reviewed~
-  [x] ~Security reviewed (no data URIs, check for nonce leak)~
-  [x] ~Tests reviewed (coverage, legitimacy)~

### CODE REVIEW

**Comment by corinagum** [comment]
I think this PR might resolve the following: https://github.com/microsoft/BotFramework-WebChat/issues/3825

Could you verify?

File: packages/component/src/SendBox/TextBox.js [hunk 1]
```diff
@@ -332,7 +333,10 @@ const TextBox = ({ className }) => {
       const { current } = inputElementRef;
 
       if (current) {
-        if (noKeyboard) {
+        // The "disable soft keyboard on mobile devices" logic will not work on IE11. It will cause the <input> to become read-only until next focus.
+        // Thus, no mobile devices carry IE11, we don't need to explicitly disable soft keyboard on IE11.
```
Comment (corinagum) [inline packages/component/src/SendBox/TextBox.js:337]:
```suggestion
        // Thus, no mobile devices carry IE11 so we don't need to explicitly disable soft keyboard on IE11.
```

**Review by corinagum — APPROVED** [review]
Approve pending comments :)

**Comment by compulim** [comment]
This doesn't fix #3825. Going to open another PR. 😄

**Pull request merged**